hw2/Task4.Tests/Task4Tests.fs

```diff
@@ -0,0 +1,35 @@
+module Hw2.Task4.Tests
+
+open NUnit.Framework
+open FsUnit
+
+[<Test>]
+let testUntil100 () =
+    primeNumbers |> Seq.takeWhile ((>) 100) |> should equal
+    <| seq {

```

Comment:
![image](https://github.com/TimaFrolov/programmingSem4/assets/26364714/6b52797f-0e6b-40f5-a34f-5d6f8f17d27b)

https://youtu.be/1AZA1zoP-II?t=2167